zap/src/main/java/org/parosproxy/paros/core/scanner/Alert.java

```diff
@@ -892,4 +892,37 @@ public void setSource(Source source) {
         }
         this.source = source;
     }
+
+    /**
+     * Checks if a value {@code int} is between {@value #RISK_INFO} (RISK_INFO) and {@value
+     * #RISK_HIGH} (RISK_HIGH)
+     *
+     * @return true if the checked risk ({@code int}) is in the range, false otherwise
+     * @since TODO add version
+     * @see #RISK_INFO
+     * @see #RISK_HIGH
+     */
+    public static boolean isValidRisk(int risk) {
+        if (risk >= RISK_INFO && risk <= RISK_HIGH) {
```

Comment:
- Could return the result of the expression instead of using an if.
 - Needs ZAP comment.